### PR TITLE
fix: Delegatee and add-mileage page, search-box is larger

### PR DESCRIPTION
**Pull request by rupam-seal**
### <samp>🤖 Generated by Copilot at 6ecd2aa</samp>

Improved the UI and UX of the delegated accounts feature and the location modal component. Added and modified styles in `delegated-accounts.page.scss`, `global.scss`, and `fy-location-modal.component.scss`. Applied the `dense-1` class to the toolbar in `delegated-accounts.page.html`.

Before

<div>
</img>
</img>
</div>

After

<div>
</img>
</img>
</div>

### <samp>🤖 Generated by Copilot at 6ecd2aa</samp>

> _We are the masters of the code, we shape it as we please_
> _We tweak the styles and classes, we make the toolbars dense_
> _We customize the fields and inputs, we enhance the user's sense_
> _We are the lords of the layout, we create the perfect interface_

### <samp>🤖 Generated by Copilot at 6ecd2aa</samp>

*  Add `dense-1` class to `ion-toolbar` element in delegated accounts page to reduce toolbar height and padding ([link](https://github.com/fylein/fyle-mobile-app/pull/2314/files?diff=unified&w=0#diff-4fc4a39a4742884b3ffe41b280ce2d7a7251bb911b194d00cc9ba3a15e7a6aacL2-R2))
* Define and apply custom styles for search input field in delegated accounts page using `$header-input-interior` variable and `mat-form-field` element ([link](https://github.com/fylein/fyle-mobile-app/pull/2314/files?diff=unified&w=0#diff-2c14cd837a93ba75e4919844ad7e17f05f75d0e4f6c1a9a1a8f53c7ab526755fR7), [link](https://github.com/fylein/fyle-mobile-app/pull/2314/files?diff=unified&w=0#diff-2c14cd837a93ba75e4919844ad7e17f05f75d0e4f6c1a9a1a8f53c7ab526755fR19-R34))
* Define and apply custom styles for location select field in location modal component using `&--form-field` element ([link](https://github.com/fylein/fyle-mobile-app/pull/2314/files?diff=unified&w=0#diff-5051e2be6a1019d5a5ef2cc4adee1dbec803f6078902f44a4b05ec2966bb7816L76-R88))
* Define `dense-1` class in global style sheet to reduce padding and height of text field wrapper and infix elements in material design components ([link](https://github.com/fylein/fyle-mobile-app/pull/2314/files?diff=unified&w=0#diff-de6c7d3b1be29b1fbdc515c9449a7d3779efcf15c279a1e8b88a2585ba47f818R409-R418))
* Remove default padding on left and right sides of `ion-toolbar` element in global style sheet ([link](https://github.com/fylein/fyle-mobile-app/pull/2314/files?diff=unified&w=0#diff-de6c7d3b1be29b1fbdc515c9449a7d3779efcf15c279a1e8b88a2585ba47f818R878-R882))

## Clickup
https://app.clickup.com/t/85ztvp5tw

## Code Coverage
Please add code coverage here

## UI Preview
Please add screenshots for UI changes

### CODE REVIEW

File: src/global.scss [hunk 1]
```diff
@@ -406,6 +406,16 @@ ion-popover.camera-options-popover {
   }
 }
 
+.dense-1 {
```
Comment (Julias0) [inline src/global.scss:409]: what is this dense-1? If you are planning to use it on title - you can name it like fy-title

File: src/global.scss [hunk 2]
```diff
@@ -865,6 +875,11 @@ ref - https://stackoverflow.com/a/31162426
   }
 }
 
+ion-toolbar {
```
Comment (Julias0) [inline src/global.scss:878]: We shouldnt do this. This will effect all toolbars and cannot be overridden. Why is this needed?

Comment (Julias0) [reply src/global.scss:878]: You should atleast add it to a class that you are defining if you need to use this

File: src/app/shared/components/fy-location/fy-location-modal/fy-location-modal.component.scss [hunk 3]
```diff
@@ -105,6 +115,8 @@
     margin-bottom: 0;
     border-bottom: 1px solid $border-grey;
     margin-top: calc(env(safe-area-inset-top));
+    padding-left: 0px;
+    padding-right: 0px;
```
Comment (Chethan-Fyle) [inline src/app/shared/components/fy-location/fy-location-modal/fy-location-modal.component.scss:119]: Remove `px` when the vaule is 0. That's the convention we use.

File: src/global.scss [hunk 4]
```diff
@@ -406,6 +406,16 @@ ion-popover.camera-options-popover {
   }
 }
 
+.mat-form-field-container {
+  .mat-mdc-text-field-wrapper.mdc-text-field--outlined .mat-mdc-form-field-infix {
+    padding-top: 0px;
+    padding-bottom: 0px;
```
Comment (Chethan-Fyle) [inline src/global.scss:412]: Remove `px`.

**Review by Chethan-Fyle — APPROVED** [review]

**Review by Julias0 — APPROVED** [review]

File: src/app/fyle/delegated-accounts/delegated-accounts.page.scss [hunk 5]
```diff
@@ -4,6 +4,7 @@ $header-input: #f8f8f8;
 $org-thumb: #f8f8f8;
 $org-thumb-header: #333;
 $org-thumb-text: #666;
+$header-input-interior: #eeeeee;
```
Comment (Dimple16) [inline src/app/fyle/delegated-accounts/delegated-accounts.page.scss:7]: we shouldn't be adding colors here

Comment (Dimple16) [reply src/app/fyle/delegated-accounts/delegated-accounts.page.scss:7]: please use the `theme/colors.scss`

File: src/app/shared/components/fy-location/fy-location-modal/fy-location-modal.component.scss [hunk 6]
```diff
@@ -73,8 +73,18 @@
 
   &--form-field {
     @extend %fy-select-mat-form-field;
-    .mat-form-field-wrapper {
-      box-shadow: rgba(0, 0, 0, 0.1) 0px 4px 8px 0px !important;
+    ::ng-deep .mat-mdc-text-field-wrapper {
```
Comment (Dimple16) [inline src/app/shared/components/fy-location/fy-location-modal/fy-location-modal.component.scss:76]: qq, for all the select input modals we use a common search bar.
Can we not use the same and just fix the spacing if required?

**Review by Dimple16 — CHANGES_REQUESTED** [review]
added comments

**Pull request closed without merging**